service/recipes.go

```diff
@@ -26,7 +26,11 @@ func FindByRecipeId(c *gin.Context) {
 
 func CreateRecipes(data []byte) bool {
 	var insert entity.ResponseBody
-	json.Unmarshal(data, &insert)
+	err := json.Unmarshal(data, &insert)
+
+	if err != nil {
+		return false
+	}
```

Comment:
We usually reduce scope of variables with only single error.

```go
if err := foo(); err != nil {
        // TODO: error handling
}
```

[Ref here](https://github.com/uber-go/guide/blob/master/style.md#reduce-scope-of-variables)

> btw, returning a bool is not a good idea for go error handling, try to wrap the error and return it.

Reply:
ok